op-service/eth/blob.go

```diff
@@ -131,3 +272,9 @@ func (b *Blob) Clear() {
 		b[i] = 0
 	}
 }
+
+func clearBuffer(buffer []byte) {
```

Comment:
you can get rid of this once you get rid of the clearBuffer calls as suggested above.